extensions/kubernetes/deployment/src/main/java/io/quarkus/kubernetes/deployment/KubernetesDeployer.java

```diff
@@ -48,7 +48,7 @@ public void deploy(KubernetesClientBuildItem kubernetesClient,
 
         if (!containerImage.isPresent()) {
             LOG.warn(
-                    "A Kubernetes deployment was requested by no extension was found to build a container image. Consider adding one of following extensions: \"quarkus-container-image-jib\", \"quarkus-container-image-docker\" or \"quarkus-container-image-s2i\",");
+                    "A Kubernetes deployment was requested but no extension was found to build a container image. Consider adding one of following extensions: \"quarkus-container-image-jib\", \"quarkus-container-image-docker\" or \"quarkus-container-image-s2i\",");
```

Comment:
```suggestion
                    "A Kubernetes deployment was requested but no extension was found to build a container image. Consider adding one of following extensions: \"quarkus-container-image-jib\", \"quarkus-container-image-docker\" or \"quarkus-container-image-s2i\".");
```

Reply:
This was already changed in one of the commits

Reply:
Oh sorry, I just saw what you meant.